### PR TITLE
{bio}(foss/2015b) SAMtools 0.1.20, Cufflinks 2.2.1(REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**

### CODE REVIEW

File: easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = [
+    'https://github.com/samtools/samtools/archive',
+]
+sources = [
+    '%(version)s.tar.gz',
+]
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb]: single-line is fine here

```python
source_urls = ['https://github.com/samtools/samtools/archive']
sources = ['%(version)s.tar.gz']
```

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+description = """Transcript assembly, differential expression, and differential regulation for RNA-Seq"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('Boost', '1.60.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb]: there are existing easyconfigs that use Boost 1.55, so maybe we need a versionsuffix?

File: easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+dependencies = [
+    ('ncurses', '5.9'),
+    ('zlib', '1.2.8'),
+]
+
+parallel = 1
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb]: other SAMtools v0.x easyconfigs don't have this, so no need for it?

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb]: @nathanhaigh: this is still 'open'?

Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb]: ping?

**Comment by boegel** [comment]
wth happened to the diff here?

I smell a screwup by GitHib... @nathanhaigh: can you sync this branch up with `develop` to 'reset' this?

**Comment by nathanhaigh** [comment]
@boegel - I can't figure out how to fix this - I'm going to delete the branch and start it again!

**Comment by nathanhaigh** [comment]
New PR in #2599

**Pull request closed without merging**